simtools/db/db_handler.py

```diff
@@ -1039,8 +1039,14 @@ def add_new_parameter(
             db_entry["instrument"] = names.validate_telescope_name(telescope)
         elif "sites" in collection_name:
             db_entry["instrument"] = names.validate_site_name(site)
+        # TODO: generalize later
+        elif "calibration_devices" in collection_name:
```

Comment:
Do you want to add one single test for this in test_adding_new_parameter_db in test_db_handler? A single line is probably enough.

Reply:
Added.